server/sonar-webserver/src/main/java/org/sonar/server/platform/web/StaticResourcesServlet.java

```diff
@@ -138,13 +138,13 @@ CoreExtensionRepository getCoreExtensionRepository() {
     }
 
     @CheckForNull
-    InputStream openPluginResourceStream(String pluginKey, String resource, PluginRepository pluginRepository) throws Exception {
```

Comment:
There are some tests that are expecting this method to support a throw of Exception, they should be adapted

Reply:
I'm really sorry. :[
The tests are fixed now.
I'll take care about it when I submit new suggestions.

Reply:
I thought better of it and I chose not to touch this class.

For me it doesn't make much sense to throw the exception, but I don't understand the full flow, so I will respect whoever implemented it, (:

Reply:
Does it make sense to you @leo-geoffroy-sonarsource?